internal/controllers/ingresses.go

```diff
@@ -117,6 +117,43 @@ func (r *Reconciler) createOrUpdateIngress(ctx context.Context, ingress *netv1.I
 	return ingress, nil
 }
 
+func configureCoreIngress(cr *model.CryostatInstance) *operatorv1beta1.NetworkConfiguration {
```

Comment:
Ahh I forgot. These functions are just declared here. They need to be called:

https://github.com/cryostatio/cryostat-operator/blob/3fe4b93719b322972edfb45982745a5a3efef4fd/internal/controllers/ingresses.go#L46

https://github.com/cryostatio/cryostat-operator/blob/3fe4b93719b322972edfb45982745a5a3efef4fd/internal/controllers/ingresses.go#L69

Reply:
That would be mean another set of tests in Ingress to work on.